services/core-web/package.json

```diff
@@ -184,7 +186,7 @@
       "^@/(.*)$": "<rootDir>/src/$1",
       "^@common/(.*)$": "<rootDir>/common/$1",
       "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga|pdf)$": "<rootDir>/src/assetsTransformer.js",
-      "\\.(css|less)$": "<rootDir>/src/assetsTransformer.js",
+      "\\.(css|less|scss)$": "<rootDir>/src/assetsTransformer.js",
```

Comment:
jest can't parse files that import scss without adding them here. Did both MS and Core even though MS didn't need it yet.